image-builder/builder-core/src/main/java/io/hyscale/builder/core/models/BuildContext.java

```diff
@@ -84,8 +85,16 @@ public void setImageRegistry(ImageRegistry imageRegistry) {
 		this.imageRegistry = imageRegistry;
 	}
 
+	public ImageRegistry getPullImageRegistry() {
```

Comment:
PullRegistry is commonly used . I think that makes more sense.
The other image registry you can change it PushRegistry